lms/templates/courseware/courseware-chromeless.html

```diff
@@ -61,6 +61,10 @@
 
 </%block>
 
+<%block name="skip_links">
```

Comment:
Please remove lines 64-67 and 95-96. SkipNav links will need to operate outside the WebView.
But the change to the <main> element -- removing tabindex="-1" -- seems appropriate, particularly if it fixes this weird TalkBack issue. Thanks!